opentech/apply/funds/templates/funds/applicationsubmission_simplified_detail.html

```diff
@@ -0,0 +1,58 @@
+{% extends "base-apply.html" %}
+
+{% load wagtailcore_tags statusbar_tags %}
+
+{% block title %}{{ object.title }}{% endblock %}
+
+{% block body_class %}{% endblock %}
+
+{% block content %}
+{% if object.round.specific.is_sealed %}
+<ul class="messages">
+    {% include "includes/message_item.html" with message="This submission is sealed " tag="warning" close=False %}
+</ul>
+{% endif %}
+<div class="admin-bar">
+    <div class="admin-bar__inner">
+        <h1 class="beta heading heading--no-margin heading--bold">{{ object.title }}</h1>
+        <h5 class="heading heading--meta">
+            <span>{{ object.stage }}</span>
+            <span>{% if public_page %}<a class="link--transparent link--underlined" href="{% pageurl public_page %}" >{{ object.page }}</a>{% else %}{{ object.page }}{% endif %}</span>
+            {% if request.user.is_apply_staff and object.round %}
+                <span><a class="link--transparent link--underlined" href="{% url 'apply:rounds:detail' pk=object.round.pk %}">{{ object.round }}</a></span>
+            {% else %}
+                <span>{{ object.round }}</span>
+            {% endif %}
+            {% if request.user.is_apply_staff %}
+                <span>Lead: <a class="link--transparent link--underlined" href="{% url 'apply:submissions:list' %}?lead={{ object.lead.id }}">{{ object.lead }}</a></span>
+            {% else %}
+                <span>Lead: {{ object.lead }}</span>
+            {% endif %}
+        </h5>
+        {% status_bar object.workflow object.phase request.user author=object.user same_stage=True %}
+    </div>
+</div>
+
+<div class="wrapper wrapper--large wrapper--tabs js-tabs-content">
+    <div class="wrapper wrapper--sidebar">
+            <article class="wrapper--sidebar--inner">
+                <header class="heading heading--submission-meta heading-text zeta">
+                    <span>Submitted: <strong>{{ object.submit_time.date }} by {{ object.user.get_full_name }}</strong></span>
+                    <span>Last edited: <strong>{{ object.live_revision.timestamp.date }} by {{ object.live_revision.author }}</strong></span>
+                </header>
+
+                {% include "funds/includes/rendered_answers.html" %}
```

Comment:
as per #1372 You should not be included the file fields in the answers - they will be coming from the supporting documents. 

This method will return all the non-file answers.
https://github.com/OpenTechFund/opentech.fund/blob/767de79cc46550939055576e7c554054e1d3eabc/opentech/apply/funds/models/mixins.py#L222-L227

The current implementation of `rendered_answers.html` uses `output_answers` - i think its probably best if we dont use the included template at all because it has links in there:
https://github.com/OpenTechFund/opentech.fund/blob/767de79cc46550939055576e7c554054e1d3eabc/opentech/apply/funds/models/mixins.py#L230-L232

Reply:
Fixed and rebased.